ppr-ui/src/components/unitNotes/UnitNoteContentInfo.vue

```diff
@@ -120,8 +129,23 @@ export default defineComponent({
   components: {
     BaseAddress
   },
-  setup () {
+  setup (props) {
     const { isNoticeOfCautionOrRelatedDocType } = useMhrUnitNote()
+
+    const localState = reactive({
+      separatedRemarks: computed(() : string[] | null => {
+        if (props.note.documentType === UnitNoteDocTypes.CONTINUED_NOTE_OF_CAUTION && !props.note?.expiryDateTime) {
```

Comment:
very small thing but you may want to optional chain the first instance of `props.note`. if its undefined, it may not reach the second instance.

Reply:
Given that note prop is required for this component, is it necessary to add the optional chaining?

![image](https://github.com/bcgov/ppr/assets/77707952/94ed46a7-c54f-456b-be27-1afd62be0e38)

Reply:
Sounds good